app/quest-boost/[boostId]/page.tsx

```diff
@@ -0,0 +1,193 @@
+"use client";
+
+import React, { useEffect, useState } from "react";
+import styles from "../../../styles/questboost.module.css";
+import { Call, CallDetails } from "starknet";
+import {
+  getBoostById,
+  getQuestBoostClaimParams,
+  getQuestParticipants,
+  getQuestsInBoost,
+  updateQuestBoostClaimStatus,
+} from "../../../services/apiService";
+import Quest from "../../../components/quests/quest";
+import { useRouter } from "next/navigation";
+import { QuestDocument } from "../../../types/backTypes";
+import Timer from "../../../components/quests/timer";
+import { useAccount, useContractWrite } from "@starknet-react/core";
+import Button from "../../../components/UI/button";
+import boostContractCalls from "../../../utils/calldata/boost_contract_claim";
+import { useNotificationManager } from "../../../hooks/useNotificationManager";
+import {
+  NotificationType,
+  TransactionType,
+} from "../../../constants/notifications";
+import { hexToDecimal } from "../../../utils/feltService";
+import { CDNImage } from "../../../components/cdn/image";
+
+type BoostQuestPageProps = {
+  params: {
+    boostId: string;
+  };
+};
+
+export default function Page({ params }: BoostQuestPageProps) {
+  const router = useRouter();
+  const { address } = useAccount();
+  const { boostId } = params;
+  const [quests, setQuests] = useState([] as QuestDocument[]);
+  const [boost, setBoost] = useState({} as Boost);
+  const [participants, setParticipants] = useState<number>();
+  const [calls, setCalls] = useState<CallDetails>({} as CallDetails);
```

Comment:
See my comment in the calldata utils file, you can update this to a `Call` instead of `CallDetails` (and also use singular in the variable name if there is only one call ?)

Reply:
done